python-frontend/src/main/java/org/sonar/python/types/TypeShed.java

```diff
@@ -283,12 +283,28 @@ private static Symbol disambiguateWithLatestPythonSymbol(Set<Symbol> alternative
     return latestPythonSymbol;
   }
 
+
+  /**
+   * Returns stub symbols to be used by SonarSecurity.
+   * Ambiguous symbols that contain all class symbols are disambiguated with latest Python version.
```

Comment:
The phrasing is a bit weird, I assume you mean "...that only contain class symbols"?

Reply:
Indeed thanks, I will change it!